Fixed issue with the copy command

srayhunter opened the pull request:
When running the `cp -r lib/ ContactList/` this is not the correct way to copy the directory into the other directory. The trailing slash needs to be removed so that the directory is copied and not the contents of the directory.

azurecla commented:
Hi __@DavidHoerster__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

srayhunter committed:
Fixed issue with the copy command

When running the `cp -r lib/ ContactList/` this is not the correct way to copy the directory into the other directory. The trailing slash needs to be removed so that the directory is copied and not the contents of the directory.
